.mkosi/mkosi.arch

```diff
@@ -41,6 +41,7 @@ BuildPackages=
         pkgconfig
         python
         python-lxml
+        python-jinja2
```

Comment:
looks like this is just `python-jinja` https://github.com/systemd/systemd-centos-ci/pull/382#discussion_r633137477